theme_bewise/views/customizations.xml

```diff
@@ -2,48 +2,84 @@
 <odoo>
 
 <!-- ======== COVER ======== -->
- <template id="s_cover" inherit_id="website.s_cover" name="Bewise s_cover">
+<template id="s_cover" inherit_id="website.s_cover" name="Be Wise s_cover">
```

Comment:
Because the real name is "Be Wise" if we look at the manifest, I thought it was a mistake :)
[https://github.com/odoo-dev/design-themes/blob/master-theme-bewise-brd/theme_bewise/__manifest__.py](url)

Reply:
Ok, view name are useless anyway so it's the same for me ;)